### PR TITLE
add compressible gabls (stable BL case)

### PR DESCRIPTION
**Pull request by yairchn**
Following our partial success last Friday I would like to hook a test to maintain these results while we work towards adding convective cases

### CODE REVIEW

File: examples/hybrid/schur_complement_W.jl [hunk 1]
```diff
@@ -44,7 +49,6 @@ function SchurComplementW(Y, transform, flags, test = false)
     @assert length(filter(is_energy_var, propertynames(Y.c))) == 1
     @assert length(filter(is_momentum_var, propertynames(Y.c))) == 1
     @assert length(filter(is_momentum_var, propertynames(Y.f))) == 1
-    @assert length(propertynames(Y.f)) == 1
```
Comment (yairchn) [inline examples/hybrid/schur_complement_W.jl:47]: @dennisYatunin are we ok with just removing this assertion?

File: examples/hybrid/driver.jl [hunk 2]
```diff
@@ -136,7 +137,7 @@ function additional_cache(Y, params, model_spec, dt; use_tempest_mode = false)
             )
         ),
         (; Δt = dt),
-        (; enable_default_remaining_tendency = isnothing(turbconv_model)),
+        (; enable_anelastic_dycore = anelastic_dycore),
```
Comment (charleskawczynski) [inline examples/hybrid/driver.jl:140]:
```suggestion
        (; enable_anelastic_dycore = model_spec.anelastic_dycore),
```
Can we avoid the closure here and store this in `model_spec`?

File: examples/hybrid/driver.jl [hunk 3]
```diff
@@ -21,6 +21,7 @@ hyperdiff = parsed_args["hyperdiff"]
 disable_qt_hyperdiffusion = parsed_args["disable_qt_hyperdiffusion"]
 turbconv = parsed_args["turbconv"]
 case_name = parsed_args["turbconv_case"]
+anelastic_dycore = parsed_args["anelastic_dycore"]
```
Comment (charleskawczynski) [inline examples/hybrid/driver.jl:24]:
```suggestion
```
Should be able to move this into `get_model_spec`

**Review by trontrytel — APPROVED** [review]
lgtm!

File: examples/hybrid/types.jl [hunk 4]
```diff
@@ -108,6 +112,7 @@ function get_model_spec(::Type{FT}, parsed_args, namelist) where {FT}
         microphysics_model = microphysics_model(parsed_args),
         forcing_type = forcing_type(parsed_args),
         turbconv_model = turbconv_model(FT, parsed_args, namelist),
+        anelastic_dycore = anelastic_dycore(parsed_args),
```
Comment (charleskawczynski) [inline examples/hybrid/types.jl:115]:
```suggestion
        anelastic_dycore = parsed_args["anelastic_dycore"],
```

File: examples/hybrid/types.jl [hunk 5]
```diff
@@ -69,6 +69,10 @@ function microphysics_model(parsed_args)
     end
 end
 
+function anelastic_dycore(parsed_args)
+    anelastic_dycore = parsed_args["anelastic_dycore"]
+end
+
```
Comment (charleskawczynski) [inline examples/hybrid/types.jl:75]:
```suggestion
```

**Review by charleskawczynski — APPROVED** [review]
LGTM. The only thing that's needed is that all of the mse tables should be set to zero when incrementing the reference counter. This is documented in the regression test [readme](https://github.com/CliMA/ClimaAtmos.jl/tree/main/regression_tests#the-basic-idea-of-how-regression-tests-work). Please open an issue if something there is unclear 🙂

**Comment by yairchn** [comment]
Bors r+

**Comment by yairchn** [comment]
Bors r-

**Comment by yairchn** [comment]
Bors r+

**Comment by yairchn** [comment]
Bors r+

**Pull request merged**